Dockerfile.ci-rp

```
@@ -2,20 +2,39 @@ ARG REGISTRY
 ARG VERSION
 
 ###############################################################################
+# Stage 1: Build the SRE Portal Assets
 # builder is responsible for all compilation and validation of the RP
 ###############################################################################
-FROM ${REGISTRY}/ubi8/go-toolset:1.20.12-5 as builder
+FROM ${REGISTRY}/ubi8/nodejs-16  as portal-build
+WORKDIR /build/portal/v2
+USER root
+
+# Copying package files and installing dependencies
+COPY portal/v2/package*.json ./
+RUN npm ci
+RUN npm audit --omit=dev # Run audit without dev dependencies
+
+# Copying the rest of the source and build
+COPY --chown=root:root portal/v2/ ./
+RUN npm run lint && \
+    npm run build || (echo "Build failed, listing current directory:" && ls -la)
```

Is the `ls` output useful here? IMO we should just let the build errors stand on their own. 

```suggestion
RUN npm run lint && npm run build
```